### ISSUE TITLE
Training basenet takes huge time

### ISSUE THREAD

**Issue by alejodosr** [vikrant7/mobile-vod-bottleneck-lstm#21]
Hi,

I am currently training basenet but I am observing that training is taking huge amount of time, not normally found in this type of datasets and models. I can achieve 0.4mAP after 3 days for training on a NVIDIA GeForce RTC 2080.

Have you been able to make a training which achieve paper's mAP? Where do you think is the issue?

Thank in advance.

**Comment by bbidong** [comment]
@alejodosr Hi,  how do you set the batch size and lr ?  And my loss is huge as follows. Are you same as me ?
![image](https://user-images.githubusercontent.com/35252861/82053527-14975180-96f0-11ea-9d5f-6a38ac20b7c0.png)

**Comment by KeepGoing2019HaHa** [comment]
@alejodosr @bbidong  Hi, I have also meet the same problem, training is slow and evaluation results is very poor. How do you guys solve these problems ?

**Comment by bbidong** [comment]
the evaluate result is lower than that in paper , because this is not the official code , so i try another vod method.------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;KeepGoing2019HaHa&quot;<notifications@github.com&gt;
发送时间:&nbsp;2020年7月20日(星期一) 中午11:55
收件人:&nbsp;&quot;vikrant7/mobile-vod-bottleneck-lstm&quot;<mobile-vod-bottleneck-lstm@noreply.github.com&gt;;
抄送:&nbsp;&quot;bbidong&quot;<384023171@qq.com&gt;;&quot;Mention&quot;<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [vikrant7/mobile-vod-bottleneck-lstm] Training basenet takes huge time (#21)

**Comment by KeepGoing2019HaHa** [comment]
@bbidong Thanks for your reply. 
Are you using code from Tensorflow? https://github.com/tensorflow/models/tree/master/research/lstm_object_detection
Can you reproduce the paper's results?

**Comment by bbidong** [comment]
I have not use this tensorflow code , maybe you can try "MEGA for Video Object Detection" which is a&nbsp;PyTorch implementation of a 2020 paper.

**Comment by KeepGoing2019HaHa** [comment]
@bbidong Thanks a lot.
